js/apps/admin-ui/cypress/support/pages/admin-ui/manage/identity_providers/social/ProviderMicrosoftGeneralSettings.ts

```diff
@@ -0,0 +1,29 @@
+import ProviderBaseGeneralSettingsPage from "../ProviderBaseGeneralSettingsPage";
+
+const tenant_id_input_test_value = "tenant_id_input_test_value";
```

Comment:
Let's use a constant case style here. Also, I don't think the name needs to be this long, considering the context it's used in.

```suggestion
const TENANT_ID_VALUE = "tenant_id_input_test_value";
```